tfjs-backend-webgl/src/backend_webgl.ts

```diff
@@ -1277,6 +1277,7 @@ export class MathBackendWebGL extends KernelBackend {
 
   getUniformLocations() {
     for (const binary of Object.values(this.binaryCache)) {
+      this.gpgpu.buildVao(binary.webGLProgram);
```

Comment:
It does not make sense to add this code within `getUniformLocations` and theoratically we should move it to a new function, like 'setVao'. However, since we have told some users to use this to utilize parallel compilation feature, making `setVao` and requiring users to call it are breaking change for them.

Reply:
Nit: Can you add a TODO here explaining this? Thanks!

Reply:
Done. Thanks!